### ISSUE TITLE
Dataset requirements

### ISSUE THREAD

**Issue by mrgloom** [rosinality/style-based-gan-pytorch#64]
Is it possible to use `img_align_celeba.zip` (1.4Gb) with images of size `178x218` to train model up to 128p? What are general requirements for dataset? should faces be aligned?

**Comment by MichaelMonashev** [comment]
> should faces be aligned?
Yes. For better quality.

**Comment by rosinality** [comment]
Yes, it will run okay as images will be center cropped. With aligned data quality will be better, but I think celebA will run well. (I have used celebA before FFHQ.)
